client/my-sites/site-settings/related-posts/index.jsx

```diff
@@ -40,8 +40,14 @@ const RelatedPosts = ( {
 						text={ translate(
 							'Automatically displays similar content (related posts) at the end of each post.'
 						) }
-						link="https://jetpack.com/support/related-posts/"
+						link="https://en.support.wordpress.com/related-posts/"
```

Comment:
Hmm, I don't think we should change these links so lightly. While it makes sense to link to the WP.com support doc for WP.com sites, it also makes sense to link to the Jetpack one for Jetpack sites. 

What do you think about conditionally defining the link URL based on the site type? 

Note: let's not forget about properly handling it for atomic sites too 😉

Reply:
While that's a nice feature to have, it might be better for another PR?

The vast majority of anyone looking at this setting will be wpcom customers/sites so if I'd have to pick one, wpcom one seems appropriate?

Reply:
Well, you said it - it might be better for another PR. Why are we even changing the URL in this PR - isn't this PR supposed to be solely for adding the explanation?

So, I suggest that we do the link change and the corresponding discussion in another PR. This has always led to jetpack, and changing it could use some feedback.

Reply:
Certainly! Valid points.

Reply:
It was changed in this PR to address that point in the original issue, but agreed that it's a change outside this PR. It's not a huge deal in terms of the information conveyed on that page either. :)